packages/sdk/source/signed-transaction.dto.ts

```diff
@@ -219,4 +223,28 @@ export class AbstractSignedTransactionData implements SignedTransactionData {
 			},
 		];
 	}
+
+	#normalizeTransactionData<T>(value: RawTransactionData): T {
+		return JSON.parse(
+			JSON.stringify(value, (key, value) => {
+				if (typeof value === "bigint") {
+					return value.toString();
+				}
+
+				if (["timestamp"].includes(key)) {
+					return DateTime.make(value).toUNIX();
+				}
+
+				if (["amount", "nonce", "fee"].includes(key)) {
+					return value.toString();
+				}
```

Comment:
@faustbrian comparing with `instanceOf` BigNumber and Date as we did before, returns always false because of the different instances created in `profiles` and `sdk` by webpack. 

As we already know the fields that require normalization (timestamp, nonce, fee, and amount), I think there is no need to make it abstract, but let me know if you think of a better alternative or if I am missing anything. Tested in the wallet and in tests and it behaves as expected.